apps/external-devices/device-descriptors/Polhemus_Libert_Latus_tracker.json

```diff
@@ -0,0 +1,13 @@
+{
+    "deviceVendor": "Polhemus",
+    "deviceName": "Liberty Latus tracker",
```

Comment:
Filename typo, and here you say Tracker, while the other trackers you typically don't. Also, I think it's actually LATUS - think it stands for something.